JotunnLib/Utils/SimpleJVL.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using Jotunn.Managers;
+using Jotunn.Entities;
+using Jotunn.Configs;
+
+namespace Jotunn.Utils
+{
+    static class SimpleJVL
```

Comment:
always specify the access modifier

Reply:
Is there a preference here whether I make it public or private?

Reply:
must be public, like the methods, otherwise people that uses Jotunn won't be able to access the SimpleJVL class and the methods inside it